cli/tests/integration/npm_tests.rs

```diff
@@ -156,6 +156,15 @@ itest!(mixed_case_package_name_local_dir {
   temp_cwd: true,
 });
 
+itest!(local_dir_resolves_symlinks {
+  args: "run -A $TESTDATA/npm/local_dir_resolves_symlinks/index.js",
+  output: "npm/local_dir_resolves_symlinks/index.out",
```

Comment:
There's an option on here to use a temporary directory here and copy in the npm/local_dir_resolves_symlinks folder (I forget what it's called, but something like temp_dir_copy or something). We can use that then delete the committed `node_modules` directory.

Reply:
Thanks, done!